src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.dao;
+
+import static core.basesyntax.db.Storage.fruits;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Test
+    public void addFruit_Ok() {
```

Comment:
I dont think we need to test `Map` methods, we have to test ours implemented logic